proxy/src/console/provider/neon.rs

```diff
@@ -278,9 +278,15 @@ impl super::Api for Api {
         // The connection info remains the same during that period of time,
         // which means that we might cache it to reduce the load and latency.
         if let Some(cached) = self.caches.node_info.get(&key) {
-            info!(key = &*key, "found cached compute node info");
-            ctx.set_project(cached.aux.clone());
-            return Ok(cached);
+            let (cached, info) = cached.take_value();
+            let info = info.map_err(|c| {
+                debug!(key = &*key, "found cached wake_compute error");
+                WakeComputeError::ApiError(ApiError::Console(*c))
+            })?;
+
+            debug!(key = &*key, "found cached compute node info");
```

Comment:
Any reason this is not an info as in line 300?

Reply:
I actually intended to change 300 to be debug :D it's no longer a needed log for prod as we have the `cold_start_info=warm_cached` flag set elsewhere for this.